test/core/transport/binder/binder_transport_test.cc

```diff
@@ -192,8 +192,8 @@ class MetadataEncoder {
   template <typename Which>
   void Encode(Which, const typename Which::ValueType& value) {
     metadata_.emplace_back(
-        std::string(Which::key()),
-        std::string(grpc_core::Slice(Which::Encode(value)).as_string_view()));
+        std::string{Which::key()},
```

Comment:
`std::string` is a class, so this should not require braces.  It's constructing a new object, not doing a cast.

Reply:
Like the above case, this is again to address this weird false positive of clang-tidy. There were two cases in total.

Reply:
I'd prefer to use a NOLINT comment instead here.  I think using parens is correct here.

Reply:
I don't have a strong opinion here but shouldn't `std::string{}` be a proper construction call?

Reply:
I believe that calls the initializer_list ctor, not the single-argument ctor.  But there's no reason we should need to do that here.  The braces are ugly and confusing, and they really don't provide any benefit in this case.

Reply:
This could be actually both a regular ctor and initializer_list ctor. Yay another C++ thing! But I'm adding NOLINT comment here.